docs/i18n/portuguese/how-to-open-a-pull-request.md

```diff
@@ -44,7 +44,7 @@ Alguns exemplos de bons títulos de PRs seriam:
 
 1. Uma vez que as edições tenham sido realizadas, será solicitado que você crie um pull request na página do GitHub do seu fork.
 
-   ![Image - Compare & pull request prompt on GitHub](https://contribute.freecodecamp.org/images/github/compare-pull-request-prompt.png)
+   ![Imagem - Comparar e solicitar o pull request no GitHub](https://contribute.freecodecamp.org/images/github/compare-pull-request-prompt.png)
```

Comment:
should "Image" be translated?

Reply:
@Sembauke I think it's OK to translate. This is used as the alt text for the image here https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=proposing-a-pull-request

Reply:
Yep this should be fine